sal_interfaces/DREAM/DREAM_Commands.xml

```diff
@@ -3,61 +3,98 @@
-    <Description>Request a full DREAM system status.</Description>
+    <EFDB_Topic>DREAM_command_dataArchived</EFDB_Topic>
+    <Description>Inform DREAM that Rubin Observatory has received and archived a data product. It is still under discussion if one or more parameters are needed or not.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>DREAM</Subsystem>
     <EFDB_Topic>DREAM_command_setWeatherInfo</EFDB_Topic>
     <Description>Send the weather information to DREAM.</Description>
     <item>
-      <EFDB_Name>weatherInfo</EFDB_Name>
-      <Description>The weather info for DREAM in JSON format.</Description>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
+      <EFDB_Name>temperature</EFDB_Name>
+      <Description>The current temperature.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>deg_C</Units>
       <Count>1</Count>
     </item>
-  </SALCommand>
-  <SALCommand>
-    <Subsystem>DREAM</Subsystem>
-    <EFDB_Topic>DREAM_command_getDataProduct</EFDB_Topic>
-    <Description>Request a data product from DREAM.</Description>
     <item>
-      <EFDB_Name>product_name</EFDB_Name>
-      <Description>The name of the DREAM data product.</Description>
-      <!-- DM-29091 TODO: Replace with an enum as soon as the data products have been defined.  -->
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
+      <EFDB_Name>humidity</EFDB_Name>
+      <Description>The current humidity.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>%</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>timestamp</EFDB_Name>
-      <Description>The TAI timestamp of observation producing the DREAM data product.</Description>
-      <IDL_Type>string</IDL_Type>
+      <EFDB_Name>wind_speed</EFDB_Name>
+      <Description>The current wind speed.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>m/s</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>wind_direction</EFDB_Name>
+      <Description>The current wind direction.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>pressure</EFDB_Name>
+      <Description>The current air pressure.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>Pa</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>rain</EFDB_Name>
+      <Description>The current rain.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>mm</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>cloudcover</EFDB_Name>
+      <Description>The current cloudcover.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>%</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>safe_observing_conditions</EFDB_Name>
+      <Description>Are the conditions safe for observing or not.</Description>
+      <IDL_Type>boolean</IDL_Type>
```

Comment:
If the condition is good enough to do the observation, why do we need to send the values such as the rain, humidity, etc.?

Reply:
Rain probably will not be used and it may disappear at some point. Humidity is needed for heating in the instruments to avoid getting too wet. Wind info is needed for opening and closing of the hatches, etc.